database/youtube_database.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE youtube_media(
```

Comment:
@djophant you're creating a table not a database, write what you do 
youre script name is not well define 

rename it with something like youtube_channel_stats.sql

Reply:
just forget to add the readme.md thanks @hadjigagny93

Reply:
@djophant dont forget to resolve the conversation after your fix commit 
no merge before resolving all conversations

Reply:
I resolve the problem